scripts/test/test-backend-cli.sh

```
@@ -51,9 +51,9 @@ echo ""
 echo "=== Test 2: Network Connectivity ==="
 echo ""
 
-# Check if we can reach stratsytem
-echo -n "1. Ping stratsytem (100.105.58.61) "
-if ping -c 1 100.105.58.61 >/dev/null 2>&1; then
+# Check if we can reach Node B
+echo -n "1. Ping Node B (<NODE_B_IP>) "
+if ping -c 1 <NODE_B_IP> >/dev/null 2>&1; then
```

Shellcheck (0.11.0)

[info] The mentioned syntax error was in this simple command.

(SC1009)